fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingScrubDangling.java

```diff
@@ -158,6 +159,8 @@ public IndexingScrubDangling(@Nonnull final IndexingCommon common,
 
         return ranges.onHasNext().thenCompose(hasNext -> {
             if (Boolean.FALSE.equals(hasNext)) {
+                Transaction tr = store.getContext().ensureActive();
+                tr.clear(indexScrubIndexRangeSubspace(store, index).range());
                 return AsyncUtil.READY_FALSE; // no more missing ranges - all done
```

Comment:
Probably worth documenting here why the range is being cleared just to avoid confusing future readers